app/assets/src/components/common/metadata_manual_input.scss

```diff
@@ -47,10 +56,10 @@
 
     .input {
       margin: $space-xxs $space-m $space-xxs 0;
-      width: 110px; // see also getColumnWidth
+      width: $metadata-input-width + 0px;
```

Comment:
Is the reason you need to add `0px` so that it will be interpreted as a "px" value?

Reply:
yes that's correct. I looked it up 📖